library/junos_commit

```diff
@@ -142,6 +150,10 @@ def main():
         # --- UNREACHABLE ---
 
     try:
+        timeout = int(args['timeout'])
+        if timeout > 0:
+            dev.timeout = timeout
+
```

Comment:
Are you suggesting changing line 120 from:
''' timeout=dict(required=False, default=0), '''

to:
''' timeout=dict(required=False, type='int', default=0), '''

If so, I'm fine with that change. However, we should also change this line to match:
https://github.com/Juniper/ansible-junos-stdlib/blob/master/library/junos_install_config#L399

Even with this change, lines 153-155 need to be:
'''
      timeout = args['timeout']
        if timeout > 0:
            dev.timeout = timeout
'''

I'm not sure if your suggesting this, but I don't support changing line 120 to:
''' timeout=dict(required=False, type='int', default=30), '''

I believe the default RPC timeout should continue to be set in only one location (in PyEZ), rather than in both PyEZ (and within multiple locations in the Ansible for Junos modules).